packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/utils/urlToFile.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+
+export const urlToFile = async url => {
+  try {
+    const res = await axios.get(url, { responseType: 'blob', timeout: 60000 });
```

Comment:
60 seconds seems a bit too much for the timeout wdyt?

Reply:
Tested with a slow 3g, 30 seconds should be enough!